CMakeLists.txt

```diff
@@ -25,6 +34,14 @@ download_dep(tinyply
         GIT_TAG point_cloud_utils
 )
 
+if(NOT NATIVE_SSE)
+  # igl has sse hardcoded, so use simde to redirect to the platforms native simd instructions.
+  download_dep(simde
```

Comment:
Could you add version tag or commit ID here so we don't get build errors in the future if there's a bug that gets introduced in the main branch of simde? 

Looks like the [latest stable release is here](https://github.com/simd-everywhere/simde/releases/tag/v0.7.2)

Reply:
AyeAye.
I'd like to add the latest commit for now, as that is the one that we are also using, so it is getting the most coverage.
I can keep an eye on this and switch to a stable release if we run into any issues while testing.